File: PRUEBA DE SUBIDA/EJERCICIOS VISUAL BASIC guias/EJERCICIO1VISUALBASIC/EJERCICIO1VISUALVASIC/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click

        Dim a As String
        a = "Programacion"

        If TextBox1.Text = a Then

            MessageBox.Show("Hola Programacion")

        Else

            MessageBox.Show("Estas Equivocado")



        End If
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Me.Close()
    End Sub
End Class

File: PRUEBA DE SUBIDA/PROFE CLAUDIA C#/ejercicios profe claudia/CONDICIONALES/boton 3/boton 3/Form1.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace boton_3
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            if (Text.Length < 10)
            {
                Text = Text + button1.Text;
            }
        }

        private void button2_Click(object sender, EventArgs e)
        {
            if (Text.Length < 10)
            {
                Text = Text + button2.Text;
            }
        }

        private void button3_Click(object sender, EventArgs e)
        {
            if (Text.Length < 10)
            {
                Text = Text + button3.Text;
            }
        }

        private void Form1_Load(object sender, EventArgs e)
        {

        }

        private void button4_Click(object sender, EventArgs e)
        {
            this.Close();
        }
    }
}